### PR TITLE
D8CORE-1012 Center images in the wysiwyg

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Centered images in the wysiwyg needed some css to make them centered.

# Need Review By (Date)
- 1/29

# Urgency
- low

# Steps to Test
1. checkout this branch
1. create a page with a wysiwyg
1. add a small image and center align it
1. after saving, validate the image is centered.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by sherakama** [comment]
Two CC fixes and the CSS selectors should be encapsulated within `.su-wysiwyg-text` so that they are not effecting everything globally.

**Comment by pookmish** [comment]
:+1:

File: src/scss/components/wysiwyg/_media.scss [hunk 1]
```diff
@@ -0,0 +1,10 @@
+@charset 'UTF-8';
+
+.su-wysiwyg-text {
```
Comment (sherakama) [inline src/scss/components/wysiwyg/_media.scss]: The more that I think of this the more I think this should be moved to the profile as the theme shouldn't know about our components.

Comment (sherakama) [reply src/scss/components/wysiwyg/_media.scss]: I'll make a ticket for later.

**Pull request merged**